setup.py

```diff
@@ -21,9 +54,8 @@
         'Programming Language :: Python',
         'Framework :: Django',
     ],
-    install_requires=[
-        'shortuuid>=0.4.3,<1.0.0'
-    ],
+    install_requires=load_requirements('requirements/requirements.txt', 'requirements/requirements-django.txt'),
```

Comment:
I wouldn't hazard to say that we have a way of doing anything in setup.py across repos yet. I've seen this pattern in other repos and brought it in here because I've found some repos where requirements.txt is kept up to date, but setup.py isn't. This seemed a reasonable solution to keep them in step. That said, it may be overkill for this repo and I'm happy to remove it if you have strong objections.